docs/tempo/website/configuration/polling.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Polling
+---
+
+The polling cycle is controlled by a number of configuration options detailed here.
+
+```
+storage:
+    trace:
+        # How often to repoll the backend for new blocks. Default is 5m
+        [blocklist_poll: <duration>] 
+
+        # Number of blocks to process in parallel during polling. Default is 50.
+        [blocklist_poll_concurrency: <int>]
+
+        # By default components will pull the blocklist from the tenant index. If that fails the component can
+        # fallback to scanning the entire bucket. Set to false to disable this behavior. Default is true.
+        [blocklist_poll_fallback: <bool>]
+
+        # Maximum number of compactors that should build the tenant index. All other components will download 
+        # the index.  Default 2.
+        [blocklist_poll_tenant_index_builders: <int>]
+```
+
+Due to the mechanics of the [tenant index]({{< relref "../operations/polling" >}}) the blocklist will stale by
```

Comment:
I think it's _to be stale_? (but I'm not a native speaker so might be wrong )
```suggestion
Due to the mechanics of the [tenant index]({{< relref "../operations/polling" >}}) the blocklist will be stale by
```

Reply:
you are correct!